pkg/build/build_test.go

```diff
@@ -13,7 +13,10 @@ import (
 	"testing"
 
 	"github.com/buildtool/build-tools/pkg"
+	"github.com/buildtool/build-tools/pkg/args"
 	"github.com/buildtool/build-tools/pkg/docker"
+	"github.com/buildtool/build-tools/pkg/version"
+
 	"github.com/docker/docker/pkg/archive"
 	"github.com/stretchr/testify/assert"
 )
```

Comment:
How come local imports are in the middle instead of in the end here? Automatic format?